tests/test_text2chem.py

```diff
@@ -0,0 +1,81 @@
+# coding=utf-8
+import json
+import os
+import unittest
+
+from text2chem.regex_parser import RegExParser
+from text2chem.parser_pipeline import ParserPipelineBuilder
+from text2chem.preprocessing_tools.additives_processing import AdditivesProcessing
+from text2chem.preprocessing_tools.chemical_name_processing import ChemicalNameProcessing
+from text2chem.preprocessing_tools.mixture_processing import MixtureProcessing
+from text2chem.preprocessing_tools.phase_processing import PhaseProcessing
+from text2chem.postprocessing_tools.substitute_additives import SubstituteAdditives
+
+
+mp = ParserPipelineBuilder() \
+    .add_preprocessing(AdditivesProcessing) \
+    .add_preprocessing(ChemicalNameProcessing) \
+    .add_preprocessing(PhaseProcessing) \
+    .add_preprocessing(MixtureProcessing)\
+    .add_postprocessing(SubstituteAdditives)\
+    .set_regex_parser(RegExParser)\
+    .build()
+
+
+class TestText2chem(unittest.TestCase):

```

Comment:
Could you also update the github workflow so that the tests are ran from github actions?

Reply:
Certainly, it's already uploaded. However, the GitHub workflow has tested text2chem v0.0.2. I am unsure how to test the version I submitted.

Reply:
OK. Once the PR will be approved, it will run. Sorry, I mised that before.